src/app/parcels/form/ParcelForm.tsx

```diff
@@ -267,6 +297,7 @@ const ParcelForm: React.FC<ParcelFormProps> = ({
                             fields={fields}
                             collectionCentresLabelsAndValues={collectionCentresLabelsAndValues}
                             packingSlotsLabelsAndValues={packingSlotsLabelsAndValues}
+                            collectionTimeSlotsLabelsAndValues={collectionSlotsLabelsAndValues}
```

Comment:
Add a drop down value of "Please select collection centre to see available times" etc. if collectionSlotLabelsAndValues are not available